patching/plugin-commands-patching/src/patchCommit.ts

```diff
@@ -49,7 +50,13 @@ export async function handler (opts: install.InstallCommandOptions & Pick<Config
   const pkgNameAndVersion = `${patchedPkgManifest.name}@${patchedPkgManifest.version}`
   const srcDir = tempy.directory()
   await writePackage(parseWantedDependency(pkgNameAndVersion), srcDir, opts)
-  const patchContent = await diffFolders(srcDir, userDir)
+
+  const includedFiles = patchedPkgManifest.files ?? []
+
+  const filteredFolderA = srcDir
+  const filteredFolderB = isEmpty(includedFiles) ? userDir : await filterIgnoredFiles(userDir, includedFiles)
```

Comment:
How about using `includedFiles.length` instead of  `isEmpty`?

Reply:
In addition, I think we can put the judgment here into the `filterIgnoreFiles` function.